src/dotnet/Program.cs

```diff
@@ -121,22 +120,39 @@ private static int ProcessArgs(string[] args)
                 ["test"] = TestCommand.Run
             };
 
+            int exitCode;            
+            var arguments = string.Empty;
```

Comment:
correct, it was being passed to ```TrackEvent``` but now we're giving that no information on arguments so it's never used. We could delete this code but it's going to need to be re-added shortly after we figure out the correct scrubbing of argument values for PII. I could leave it there and add a comment on the ```TrackEvent``` call to this effect or I could just delete this stuff if you prefer.